src/components/UpdateJournalEntryModal/UpdateJournalEntryModal.tsx

```diff
@@ -0,0 +1,247 @@
+    const [kegels, setKegels] = useState<number | string>(entry.kegels);
+    const [garlandPose, setGarlandPose] = useState<number | string>(entry.garlandPose);
+    const [prenatalVitamins, setPrenatalVitamins] = useState<boolean | null>(entry.prenatalVitamins);
+    const [probiotics, setProbiotics] = useState<boolean | null>(entry.probiotics);
+    // const [previousEntry, setPreviousEntry] = useState<{}>(entry)
+    // const updateJournalEntry = useUpdateJournalEntry();
+    const queryClient = useQueryClient();
+
+    const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        handleUpdateResults()
+        // const updatedJournalEntry = {
+        //     id: entry.id,
+        //     date: date,
+        //     waterIntake: waterIntake,
+        //     proteinIntake: proteinIntake,
+        //     exercise: exercise,
+        //     kegels: kegels,
+        //     garlandPose: garlandPose,
+        //     prenatalVitamins: prenatalVitamins,
+        //     probiotics: probiotics,
+        //     authorId: userId
+        // };
+
+        // if (JSON.stringify(updatedJournalEntry) !== JSON.stringify(previousEntry)) {
+        //     updateJournalEntry.mutate(updatedJournalEntry, {
+        //         onError: (err: any) => {
+        //             handleSubmitResults(true, err.message)
+        //         },
+        //         onSuccess: () => {
+        //             handleSubmitResults(false)
+        //         }
+        //     });
+        //     handleSubmitResults(false);
+        // } else {
+        //     handleSubmitResults(true);
+        // };
+    };
+
+    // const handleCancelUpdateEntryClick = (callback: () => void) => {
+    //     return () => {
+    //         callback()
+    //     };
+    // };
+
+    // useEffect(() => {
+    //     if (data && data.length) {
+    //         setDate(data[0].date)
+    //         setWaterIntake(data[0].waterIntake)
+    //         setProteinIntake(data[0].proteinIntake)
+    //         setExercise(data[0].exercise)
+    //         setKegels(data[0].kegels)
+    //         setGarlandPose(data[0].garlandPose)
+    //         setPrenatalVitamins(data[0].prenatalVitamins)
+    //         setProbiotics(data[0].probiotics)
+    //         setPreviousEntry(
+    //             {
+    //                 id: entryId,
+    //                 date: data[0].date,
+    //                 waterIntake: data[0].waterIntake,
+    //                 proteinIntake: data[0].proteinIntake,
+    //                 exercise: data[0].exercise,
+    //                 kegels: data[0].kegels,
+    //                 garlandPose: data[0].garlandPose,
+    //                 prenatalVitamins: data[0].prenatalVitamins,
+    //                 probiotics: data[0].probiotics,
+    //                 authorId: userId
+    //             }
+    //         )
+    //     } 
+    // }, [data, entryId, userId]);
```

Comment:
Can remove, no longer necessary with passing the entry's data through props instead of having to query for it